cmds/core/ip/ops.go

```diff
@@ -130,3 +131,66 @@ func showNeighbours(w io.Writer, withAddresses bool) error {
 	}
 	return nil
 }
+
+const (
+	defaultFmt = "default via %v dev %s proto %s metric %d\n"
+	routeFmt   = "%v dev %s proto %s scope %s src %s metric %d\n"
+)
+
+var rtProto = map[int]string{
+	unix.RTPROT_BOOT:   "boot",
+	unix.RTPROT_KERNEL: "kernel",
+	unix.RTPROT_STATIC: "static",
+	unix.RTPROT_DHCP:   "dhcp",
+}
+
+func showRoutes(inet6 bool) error {
+	var f int
+	if inet6 {
+		//TODO: implement ipv6 implementation
+		//f = netlink.FAMILY_V6
+		return fmt.Errorf("ipv6 route not implemented yet")
```

Comment:
I wonder if we should print the raw content of `/proc/net/ipv6_route` here, plus a message saying that this is not implemented yet, at least it will provide some information to the user. Any opinions? CC @hugelgupf

Reply:
Oops sorry, I just sent my draft comment. Just do what we did before. Dump the content. No need for the note.

Reply:
okay i will change back for IPv6